components/applicant/Layout.tsx

```diff
@@ -47,7 +47,7 @@ function Meta() {
 // Header
 function Header() {
   return (
-    <Center height="108px" width="100%" backgroundColor="#f4f6fc">
+    <Center height="108px" width="100%" backgroundColor="background.grey">
```

Comment:
Also need to apply background color to the `Flex` box that wraps `ApplicantGrid` above. Same with internal `Layout`